### ISSUE TITLE
Imaging of non-calibrator measurement sets timing-out and pixel size

### ISSUE THREAD

**Issue by gemmaanderson** [PaulHancock/MWA-fast-image-transients#18]
I have been trying to image two datasets of J1820: 
1207602104
1207598152

Currently the time for imaging is set to 8 hrs.
#SBATCH --time=8:00:00

The output error is:

```
> less image_1207598152.e4398108
+ wsclean -name 1207598152-2m -size 4096 4096 -abs-mem 28 -weight briggs -1 -mfsweighting -scale 32asec -pol xx,yy -minuv-l 30 -joinpolarizations -niter 20000 -threshold 0.2 1207598152.ms
slurmstepd: error: *** JOB 4398108 ON nid00092 CANCELLED AT 2018-05-22T01:53:36 DUE TO TIME LIMIT ***
slurmstepd: error: Exceeded step memory limit at some point.
slurmstepd: error: Exceeded job memory limit at some point.
```

When imaging a calibrator dataset last week, the process did not time-out. It also 

I don't think the new pixel size is being propagated through (see wsclean command above). 
The above cleaning script was generated by the following input:
`> ./bin/obs_image.sh -d 4398107 -p 16arcsec -c 1207598152`

The header of the calibrator image I made last week when running:
`> ./bin/obs_image.sh -p 16arcsec -c 1207582088`
says that CDELT2  =  0.00888888888888889 (32 arcsec)

I have not tried changing the image size.

**Comment by gemmaanderson** [comment]
Imaging seems to be completing now.

Still need a fix on propagating parameters set on the command line into the final script.

**Comment by PaulHancock** [comment]
This was fixed (by @gemmaanderson) in f5681b54e66835e803786e2d160897fd96665725
